std/typecons.d

```diff
@@ -2638,33 +2638,36 @@ unittest
 
 /**
 $(D WhiteHole!Base) is a subclass of $(D Base) which automatically implements
-all abstract member functions as throw-always functions.  Each auto-implemented
-function fails with throwing an $(D Error) and does never return.  Useful for
-trapping use of not-yet-implemented functions.
+all abstract member functions as functions that always fail. These automatically
+implemented function simply throw an $(D Error) and never return. `Whitehole` is 
```

Comment:
Also, as we're trying to make it easier to understand, there's no reason to specify that these functions are automatically implemented, we've already done that in the previous sentence.